scripts/byron-to-alonzo/mkfiles.sh

```diff
@@ -476,13 +487,14 @@ mkdir -p run
 
 for NODE in ${BFT_NODES}; do
   (
+    SPROCKET="$(sprocket "${ROOT}/${NODE}/node.sock")"
```

Comment:
Do you need to define a variable? Can you not use this inline. Likewise in the next hunk.

Reply:
Inlined!